harness/tests/determined/common/experimental/test_determined.py

```diff
@@ -123,3 +123,18 @@ def test_default_retry_doesnt_retry_allowed_status(
     with pytest.raises(errors.BadRequestException):
         client.get_model(model_resp.model.name)
     responses.assert_call_count(get_model_url, 1)
+
+
+@responses.activate
+def test_get_trial_includes_summary_metrics(make_client: Callable[[], Determined]) -> None:
```

Comment:
nit: the word "includes" doesn't seem maximally precise to me. How about "populates" or another more-focused word of your choosing?

Reply:
also didn't really like "includes" as i was writing it. i like `populates`. updated.